packages/share_plus_macos/README.md

```diff
@@ -0,0 +1,15 @@
+# share_plus_macos
+
+A new flutter plugin project.
+
+## Getting Started
+
+This project is a starting point for a Flutter
+[plug-in package](https://flutter.dev/developing-packages/),
+a specialized package that includes platform-specific implementation code for
+Android and/or iOS.
+
+For help getting started with Flutter, view our
+[online documentation](https://flutter.dev/docs), which offers tutorials,
+samples, guidance on mobile development, and a full API reference.
+
```

Comment:
Not that it matters a lot, but it looks a little nicer if it atleast says something along the lines of:
> Federated macos implementation package for `share_plus`

Reply:
good catch, thanks! we are using the same template for all platform plugin readme files